app/Util.js

```diff
@@ -13,8 +13,49 @@ export function nowStr() {
   return dayjs().format('H:mm');
 }
 
+export function isEmpty(obj) {
+
+  // null and undefined are "empty"
+  if (obj == null) return true;
+  if (obj == 'indefined') return true;
```

Comment:
This seems incorrect? do you mean `typeof obj === 'undefined'`

Reply:
Function removed